hack/gosec.sh

```diff
@@ -7,10 +7,14 @@ mkdir -p $ARTIFACTS
 echo "Run go sec in pkg"
 cd $KUBEVIRT_DIR/pkg
 
-if [ -z $GOSEC ]; then
-    gosec -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
-else
+OUT="${ARTIFACTS}/junit-gosec.xml"
+ARGS="-sort -quiet -out=$OUT -exclude-dir=testutils -fmt=junit-xml ./..."
+if [ ! -z $GOSEC ]; then
     echo "Running subset"
     echo $GOSEC
-    gosec -include=$GOSEC -sort -quiet -out=${ARTIFACTS}/junit-gosec.xml -exclude-dir=testutils -fmt=junit-xml ./...
+    ARGS="-include=$GOSEC $ARGS"
+fi
+if ! gosec $ARGS; then
+    echo "report written to" $(echo "$OUT" | sed 's/.*_out/_out/')
```

Comment:
I agree it's better to move out the gosec command and just set the args differently in the GSEC condition,
If you want to print only in the fail case, I think  you may want to add and "else" with printout of "No errors detected" or something